pkg/metrics/storage/storage_sampler.go

```diff
@@ -150,6 +150,14 @@ func (ss *Sampler) Disabled() bool {
 	return ss.Interval() <= config.FREQ_DISABLE_SAMPLING
 }
 
```

Comment:
In macos partitions don't have IO Counters related data so samples won't. But we can use Storage Sampler IOCounters for IO related information inside System Sampler. This is the reason of exporting these 👇